src/libmeasurement_kit/ooni/resources_impl.hpp

```diff
@@ -0,0 +1,200 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <fstream>
+#include <regex>
+
+#include <measurement_kit/http.hpp>
+#include <measurement_kit/ooni.hpp>
+
+#include "../common/utils.hpp"
+
+namespace mk {
+namespace ooni {
+namespace resources {
+
+using namespace mk::http;
+
+template <MK_MOCK_NAMESPACE(http, get)>
+void get_latest_release_impl(Callback<Error, std::string> callback,
+                             Settings settings, Var<Reactor> reactor,
+                             Var<Logger> logger) {
+    std::string url =
+        "https://github.com/OpenObservatory/ooni-resources/releases/latest";
```

Comment:
It would be useful, especially for testing, to in some way support replacing the value of the base URL. Maybe just putting it in a MACRO would be enough, or perhaps even exposing it in a setting.

See, for example, the https://github.com/OpenObservatory/ooni-resources.testing repository that I have been using to test new resource updates in ooniprobe.